### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by imp4287**
I added a sentence on the main page (index).

### CODE REVIEW

**Comment by imp4287** [comment]
Sorry.

**Comment by pryrt** [comment]
> Sorry.

No need to be sorry.  If you'd like to try rephrasing it, you may.  If you'd just rather I do it, that's fine, too.

File: content/_index.md [hunk 1]
```diff
@@ -10,4 +10,6 @@ Notepad++ User Manual
 
 Of course, it's not our goal to create bad documentation, but you get what we mean.
 
-**Notepad++ User Manual** is built collaboratively, and [your contribution is very welcome](https://github.com/notepad-plus-plus/npp-usermanual).
\ No newline at end of file
+**Notepad++ User Manual** is built collaboratively, and [your contribution is very welcome](https://github.com/notepad-plus-plus/npp-usermanual).
+
+If you want to check out [Notepad++](https://notepad-plus-plus.org/), you can download it from the webpage, but I guess you are there.
```
Comment (donho) [inline content/_index.md]: What's the point?

**Comment by donho** [comment]
I refer this one:
https://github.com/notepad-plus-plus/npp-usermanual/commit/8dc7c306804b9e53e1d5ff66233e137fd7352073

KISS !!

**Pull request closed without merging**